src/compressed_tensors/utils/match.py

```diff
@@ -72,8 +77,8 @@ def match_named_modules(
 
 def match_named_parameters(
     model: torch.nn.Module,
-    targets: Iterable[str],
-    ignore: Iterable[str] = tuple(),
+    targets: Optional[Iterable[str]] = None,
```

Comment:
```suggestion
    targets: Optional[Iterable[str]],
```
Suggested change
    targets: Optional[Iterable[str]] = None,
    targets: Optional[Iterable[str]]
This function doesn't really make any sense if the user doesn't have targets to match with